### DISCUSSION TITLE
Auto-show/hide sidebar by mouseover doesn't work on latest Firefox Nightly

### DISCUSSION THREAD

**Discussion by YHNdnzj** [piroor/treestyletab#3216] [Q&A] [answered]
After updating to today's Firefox Nightly, https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#auto-showhide-sidebar-by-mouseover-hover doesn't work for me anymore. Is there a way to fix this?

**Comment by YHNdnzj** [comment]
For `107.0a1.20221003.11` it does hide, but is still much wider than what I specified in `userChrome.css`.

**Reply by irvinm** [reply]
I tried it in nightly and see what you are saying.  (It works fine in the beta channel)  It looks like there is probably a defect in nightly that needs to be fixed.  I think you are going to have to wait until this is resolved on the Firefox side.

I see Dev Ed around 50px minimum and Nightly around 210px minimum ... both seem to expand to the same size.

![image](https://user-images.githubusercontent.com/979729/193908140-9559fec0-3d82-4637-aa97-326dba24175c.png)

**Reply by YHNdnzj** [reply]
Well, still not fixed on `109.0a1.20221202.23`...

**Reply by piroor** [reply]
Removing `xulstore.json` in your profile to clear old size of Firefox's UI wrongly stored and restored possibly solve the problem. Could you try that?  @YHNdnzj

**Reply by YHNdnzj** [reply]
@piroor I didn't find `xulstore.json`, but only a directory named `xulstore` containing `data.safe.bin`.

Tried removing it and it got worse with a weird white border...

![2](https://user-images.githubusercontent.com/29118423/206705451-8d285bec-807e-4088-a11e-90066e161b3e.png)

**Comment by YHNdnzj** [comment, marked as answer]
https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules/_compare/ca0e4ecca608bba6149baa5fecc54aff2dcf1749...d158e4df061fa916002b39de5c23e6dfecc7dd4b

Oops, I totally ignored the wiki update... Adding `!important` fixed this for me :)
